frontend.rst

```diff
@@ -93,6 +86,7 @@ Other Front-End Articles
     :hidden:
     :glob:
 
+    frontend/assetic
```

Comment:
Looks like this does not help. I also forgot to remove one paragraph. Both issues should be fixed with #10059.